sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -2505,96 +2449,182 @@ def __getitem__(self, key):
     return self.attributes[key].__get__(self, WriteResult)
 
 
-def _default_io_expansion_service(append_args=None):
-  return BeamJarExpansionService(
-      'sdks:java:io:google-cloud-platform:expansion-service:build',
-      append_args=append_args)
-
-
 class StorageWriteToBigQuery(PTransform):
```

Comment:
I'd add some commentary that dynamic schemas are not yet supported

Reply:
Done